web-portal/store/utils/getToken.js

```diff
@@ -0,0 +1,8 @@
+const TOKEN_KEY = '_token.local'
+
+export default function getToken($auth) {
+  // according to docs this should be much simpler but this is the only way I seem to be able to get the token
+  // (https://github.com/nuxt-community/auth-module/issues/844)
+
+  return $auth.$storage.getState(TOKEN_KEY)
+}
```

Comment:
`utils` probably needs to be a top-level directory, and not in `store/`. Per the [docs](https://nuxtjs.org/docs/directory-structure/store), every `.js` file in `store/` is treated as a Vuex store module.

Reply:
:-( OK, good catch. I'll move it. I'm not a huge fan of how nuxt makes us spread out our implementation details. NextJs has the same problem. It seems like they could give you an annotation to let you exclude some things